nginx.conf

```diff
@@ -34,6 +34,10 @@ http {
     listen <%= ENV["PORT"] %>;
     server_name localhost;
 
+    location = /manuals/logit-io.html {
+        return 301 https://reliability-engineering.cloudapps.digital/manuals/logit-io.html
```

Comment:
have you tested this? it looks like it's missing a semicolon.

not sure how to test nginx changes - push to a throwaway app?